### ISSUE TITLE
Only validation loss tracked

### ISSUE THREAD

**Issue by skinnider** [skinniderlab/CLM#253]
In `train_models_RNN.py`, the `training_step` function tracks both the loss and the validation loss; however, only the validation loss is logged: https://github.com/skinniderlab/CLM/blob/master/src/clm/commands/train_models_RNN.py#L182 (`loss` and `validation_loss` passed into `track_loss` are identical)

As a result, the "training loss" and "validation loss" logged to the *loss.csv.gz files are the same at every step, e.g.:

```
epoch,minibatch,outcome,value
1,100,training loss,28.462129592895508
1,100,validation loss,28.462129592895508
1,179,training loss,19.119808197021484
1,179,validation loss,19.119808197021484
2,200,training loss,16.753406524658203
2,200,validation loss,16.753406524658203
2,300,training loss,14.778688430786133
2,300,validation loss,14.778688430786133
```

This is not critical but it'd be nice to track both losses.
